### PR TITLE
added event controller with post method

### PR DESCRIPTION
**Pull request by RBarnabass**

### CODE REVIEW

File: src/main/java/com/softserve/actent/controller/EventController.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    private final EventConverter eventConverter;
+
+    @Autowired
+    public EventController(EventService eventService, EventConverter eventConverter) {
+        this.eventService = eventService;
+        this.eventConverter = eventConverter;
+    }
+
+    @PostMapping(value = "/events")
+    public ResponseEntity<Long> addEvent(@RequestBody EventCreationDto eventCreationDto) {
```
Comment (md1guy) [inline src/main/java/com/softserve/actent/controller/EventController.java:29]: Implement your controller methods without ResponseEntity, check out this class: https://github.com/lv-381-java/Actent/blob/develop/src/main/java/com/softserve/actent/controller/TagController.java

Comment (md1guy) [reply src/main/java/com/softserve/actent/controller/EventController.java:29]: Also now we are using IdDto as we want JSON response, but not just pure Long/String. Maybe there is any other solution, but until this moment we were stuck with this approach.

**Review by md1guy — APPROVED** [review]

File: src/main/java/com/softserve/actent/controller/EventController.java (hunk 1 above)
Comment (md1guy) [reply src/main/java/com/softserve/actent/controller/EventController.java:29]: Approved by accident, in fact am asking for changes.

**Review by md1guy — CHANGES_REQUESTED** [review]

**Review by Senchakmen — DISMISSED** [review]

File: src/main/java/com/softserve/actent/controller/EventController.java (hunk 1 above)
Comment (RBarnabass) [reply src/main/java/com/softserve/actent/controller/EventController.java:29]: Only after team decision or after Ihor approve it!

Comment (md1guy) [reply src/main/java/com/softserve/actent/controller/EventController.java:29]: Whole team used IdDto's everywhere, it counts as team decision? Also I discussed it with Viktoriya and she 
acknowledged this as maybe not so good, but solution.

And what's about ResponseEntity - Ihor mentioned this twice already in various PRs.

**Review by md1guy — CHANGES_REQUESTED** [review]

File: src/main/java/com/softserve/actent/controller/EventController.java (hunk 1 above)
Comment (RBarnabass) [reply src/main/java/com/softserve/actent/controller/EventController.java:29]: ok

**Review by Senchakmen — APPROVED** [review]

**Pull request closed without merging**